app/controllers/subscription_authentication_controller.rb

```diff
@@ -25,7 +25,16 @@ def authenticate
       frequency: @frequency,
     )
 
-    redirect_to subscription_path(topic_id: @topic_id, frequency: @frequency)
+    redirect_to subscription_complete_path(topic_id: @topic_id, frequency: @frequency)
+  end
+
+  def complete
+    topic_id = params.require(:topic_id)
+    @frequency = params.require(:frequency)
+
+    @title = email_alert_api
+      .get_subscriber_list(slug: topic_id)
+      .to_h.dig("subscriber_list", "title")
```

Comment:
I don't think the 'to_h' is strictly necessary

Reply:
Unfortunately we do: turns out it's a `GdsApi::Response`, which doesn't support `fetch` or `dig` directly :-(.